rules/cross-platform/guided_onboarding_sample_rule.toml

```diff
@@ -15,18 +15,8 @@ enabled = false
 false_positives = [
     "This rule is not looking for threat activity. Disable the rule if you're already familiar with alerts.",
 ]
-from = "now-24h"
-index = [
-    "apm-*-transaction*",
-    "auditbeat-*",
-    "endgame-*",
-    "filebeat-*",
-    "logs-*",
-    "packetbeat-*",
-    "traces-apm*",
-    "winlogbeat-*",
-    "-*elastic-cloud-logs-*",
```

Comment:
It needs setup, not default. Also, the user is not recommended to ship those logs to the same instance.

![image](https://github.com/elastic/detection-rules/assets/26856693/d6b5e80e-6bd4-488b-8402-4cab1e85996f)